lib/sqlalchemy/orm/mapped_collection.py

```diff
@@ -76,25 +94,36 @@ class _SerializableColumnGetterV2(_PlainColumnGetter):
 
     __slots__ = ("colkeys",)
 
-    def __init__(self, colkeys):
+    def __init__(self, colkeys: List[Tuple[str, Optional[str]]]) -> None:
         self.colkeys = colkeys
         self.composite = len(colkeys) > 1
 
-    def __reduce__(self):
-        return self.__class__, (self.colkeys,)
+    def __reduce__(
+        self,
+    ) -> Tuple[
+        Type[_SerializableColumnGetterV2], Tuple[List[Tuple[str, Any]]]
+    ]:
+        return self.__class__, (self.colkeys,)  # type: ignore
```

Comment:
Somehow mypy is happy about this, even with all ignores in the module removed

Reply:
@jazzthief Thanks! I've removed that ignore